### ISSUE TITLE
Cannot plot single-qubit states using plot_bloch_multivector

### ISSUE THREAD

**Issue by frankharkins** [Qiskit/qiskit-terra#4941]
### Information

- **Qiskit Terra version**: `0.15.1`
- **Python version**: `3.7.7`
- **Operating system**: OSX Catalina `10.15.6`

### What is the current behavior?

`plot_bloch_multivector` fails if the state is a single-qubit state.

### Steps to reproduce the problem
```
from qiskit.visualization import plot_bloch_multivector
plot_bloch_multivector([1,0])
```
Output:
```
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-2-1670ff5274a8> in <module>
      2 from math import pi
      3 from qiskit.visualization import plot_bloch_multivector
----> 4 plot_bloch_multivector([1,0])

/usr/local/anaconda3/lib/python3.7/site-packages/qiskit/util.py in wrapper(*args, **kwargs)
    108             if kwargs:
    109                 _rename_kwargs(func.__name__, kwargs, kwarg_map)
--> 110             return func(*args, **kwargs)
    111         return wrapper
    112     return decorator

/usr/local/anaconda3/lib/python3.7/site-packages/qiskit/visualization/state_visualization.py in plot_bloch_multivector(state, title, figsize, rho)
    269                           'matplotlib".')
    270     # Data
--> 271     bloch_data = _bloch_multivector_data(state)
    272     num = len(bloch_data)
    273     width, height = plt.figaspect(1/num)

/usr/local/anaconda3/lib/python3.7/site-packages/qiskit/visualization/utils.py in _bloch_multivector_data(state)
    311     for i in range(num):
    312         paulis = PauliTable(np.zeros((3, 2 * (num - 1)), dtype=np.bool)).insert(
--> 313             i, pauli_singles, qubit=True)
    314         bloch_state = [np.real(np.trace(np.dot(mat, rho.data))) for mat in paulis.matrix_iter()]
    315         bloch_data.append(bloch_state)

/usr/local/anaconda3/lib/python3.7/site-packages/qiskit/quantum_info/operators/symplectic/pauli_table.py in insert(self, ind, value, qubit)
    323 
    324         # Column insertion
--> 325         if ind > self.num_qubits:
    326             raise QiskitError("Index {} is greater than number of qubits"
    327                               " in the PauliTable ({})".format(ind, self.num_qubits))

TypeError: '>' not supported between instances of 'int' and 'NoneType'
```

### What is the expected behavior?

Correctly plotted Bloch sphere.

**Comment by Hirmay** [comment]
It's weird as it worked for me
![image](https://user-images.githubusercontent.com/56473003/90505049-9ffd0480-e16f-11ea-9439-1c938448266f.png)

**Comment by frankharkins** [comment]
@Hirmay do you have terra 0.15.1 installed?

**Comment by Hirmay** [comment]
Oh my bad, I was just going to delete that as I realized that mine is older version.

**Comment by frankharkins** [comment]
That's okay. I am not part of the terra team though so I cannot assign.

**Comment by Hirmay** [comment]
Okay so @1ucian0 can you assign me this issue?

**Comment by 1ucian0** [comment]
assinged!

**Comment by Hirmay** [comment]
Thank you @1ucian0

**Comment by Hirmay** [comment]
I have found what is causing the error but I need help to solve it @1ucian0

**Comment by 1ucian0** [comment]
Great! Do you want to PR and we discuss there? Or do you prefer explain here first?

**Comment by Hirmay** [comment]
Fixes #4941

**Comment by Hirmay** [comment]
I have created a PR you can check that out and we can hopefully fix the bug.

**Comment by frankharkins** [comment]
@1ucian0 should I close this issue now a fix has been merged?

**Comment by 1ucian0** [comment]
Closed via #4948

**Comment by alyabouzaid** [comment]
How can I access the fixed code? Do I have to wait until the next release? I'm stuck in this error now and it would be great if I can know how manage this error now if possible.

**Comment by 1ucian0** [comment]
The fix is tagged to be included in 0.15.2, to be released soon™.

If you need the fix right now, you can install the master branch code.

**Comment by alyabouzaid** [comment]
Sounds good, thank you!
